pkg/ottl/ottlfuncs/README.md

```diff
@@ -14,14 +14,14 @@ See the component-specific guides for how each uses error mode:
 
 ## Functions
 
-Functions are the way that components that use OTTL transform telemetry.
+Functions are the way that components that use OTTL to transform telemetry.
```

Comment:
```suggestion
Functions are what OTTL uses to transform telemetry.
```
that extra `to` didn't sound right in the original sentence.

Reply:
This reads much better, thanks.